sumpy/expansion/local.py

```diff
@@ -405,6 +410,14 @@ def loopy_translate_from(self, src_expansion):
             f"A direct loopy kernel for translation from "
             f"{src_expansion} to {self} is not implemented.")
 
+    def get_loopy_evaluator(self, kernels: Sequence[Kernel]) -> lp.TranslationUnit:
```

Comment:
This should be named consistently with the above. How about `loopy_evaluate`?

Reply:
Currently we have a few names that are inconsistent with each other.
```
get_loopy_expansion_formation  (for p2e)
get_loopy_evaluator  (e2p)
loopy_translate_from (m2l)
preprocess_multipole_loopy_knl
postprocess_local_loopy_knl
```
Any suggestions on which to use?

Reply:
I think I would prefer `loopy_purpose_of_thing`. No `get`. But the name *should* indicate whether optimizations are also being returned.

Reply:
Btw, I would not be opposed to cleaning up the inconsistencies in this PR.

Reply:
`loopy_evaluate_with_optimizations`?

Reply:
That doesn't work: it reads as "evaluate with optimizations", which is not what this is. Maybe making it `loopy_noun_and_optimizations`, so `loopy_evaluator_and_optimizations` might be better?